### ISSUE TITLE
Image alt attributes

### ISSUE THREAD

**Issue by ghost** [laszlar/prj-rev-bwfs-tea-cozy#4]
https://github.com/laszlar/prj-rev-bwfs-tea-cozy/blob/master/teacozy/index.html#L36

Make sure to give your `img` tags an `alt` attribute describing the image.

This helps improve the accessibility of your site when it's unable to render these images, like when visually impaired users use readers or other devices that just don't display images but describe them instead.
